dashboard/templates/assignments.html

```diff
@@ -87,11 +93,10 @@ <h4>Assignments Due By Date</h4>
 </div>
 
 </div>
+<div class ="error"></div>
 {% endblock %}
 
 {% block js %}
-<script src="{% static 'underscore/underscore-min.js' %}"></script>
-<script src="/static/js/app.js"></script>
-<script src="/static/js/controllers.js"></script>
-<script src="/static/js/factories.js"></script>
+<script src="{% static 'd3-v4/build/d3.min.js' %}"></script>
```

Comment:
You said you wanted to use v3? Is v4 okay?

Reply:
SOrry for the confusion, I need v4!